src/generators/cfml.ts

```diff
@@ -0,0 +1,113 @@
+import * as util from "../util.js";
+import type { Request } from "../util.js";
+
+import jsesc from "jsesc";
+
+const quote = (str: string): string => {
+  return jsesc(str, { quotes: "single" }).replace(/"/g, '""');
+};
+
+export const _toCFML = (request: Request): string => {
+  let cfmlCode = "";
+
+  cfmlCode += "httpService = new http();\n";
+  cfmlCode += 'httpService.setUrl("' + quote(request.url as string)  + '");\n';
+  cfmlCode += 'httpService.setCharset("utf-8");\n';
+
+  cfmlCode +=
+    'httpService.setMethod("' + quote(request.method.toUpperCase()) + '");\n';
+
+  if (request.headers) {
+    for (const [headerName, headerValue] of request.headers) {
+      cfmlCode +=
+        'httpService.addParam(type="header", name="' +
+        quote(headerName) +
+        '", value="' +
+        quote(headerValue as string) +
+        '");\n';
+    }
+  }
+
+  if (request.timeout) {
+    cfmlCode +=
+      'httpService.setTimeout("' + (parseInt(request.timeout) || 0) + '");\n';
+  }
+
+  if (request.proxy) {
+    const proxyPart = (request.proxy as string).match(/:([0-9]{2,})/);
+    const proxyPort = proxyPart && proxyPart.length > 1 ? proxyPart[1] : "";
+    const proxy = request.proxy.replace(":" + proxyPort, "");
+
+    cfmlCode += 'httpService.setProxyServer("' + quote(proxy) + '");\n';
+    cfmlCode += 'httpService.setProxyPort("' + quote(proxyPort) + '");\n';
+
+    if (request.proxyAuth) {
+      const proxyauth = request.proxyAuth.split(":");
+      cfmlCode += 'httpService.setProxyUser("' + quote(proxyauth[0]) + '");\n';
+      cfmlCode += 'httpService.setProxyPassword("' + quote(proxyauth[1]) + '");\n';
+    }
+  }
+
+  if (request.cookies) {
+    for (const [headerName, headerValue] of request.cookies) {
+      cfmlCode +=
+        'httpService.addParam(type="cookie",name="' +
+        quote(headerName) +
+        '", value="' +
+        quote(headerValue) +
+        '");\n';
+    }
```

Comment:
```suggestion
    }
    deleteHeader(request, 'cookie');
```

you'll also need to import that up top

Reply:
Added into the new PR